components/molecule/breadcrumb/src/index.js

```diff
@@ -6,6 +6,7 @@ import ChevronRight from '@s-ui/react-icons/lib/Chevronright'
 import {BASE_CLASS, breadcrumbClassName, isFunction} from './settings.js'
 
 const BreadcrumbBasic = ({
+  ariaLabelText,
```

Comment:
We can use the existing label as default:
```suggestion
  ariaLabelText = "breadcrumb",
```

Reply:
Yesterday, talking with @andresin87, he told me not to put values for defects in the component, only in the Demo.